File: backend/src/main/java/tfg/proyecto/TFG/dtos/DTOUsuarioReportado.java
package tfg.proyecto.TFG.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class DTOUsuarioReportado {
	
	private Long id;
	private Boolean reportado;
	private String email;
	private String nombre; 
}

File: appMovilShowPass/app/src/main/java/com/example/appmovilshowpass/data/remote/dto/DTOUsuarioReportado.kt
package com.example.appmovilshowpass.data.remote.dto

import com.example.appmovilshowpass.model.TarjetaBancaria
import com.google.gson.annotations.SerializedName
import java.math.BigDecimal
import java.time.LocalDate

data class DTOUsuarioReportado(
    val id: Long = 0,
    val nombre: String = "",
    val email: String = "",
    val reportado: Boolean = false,
)

fun DTOUsuarioReportado.toUsuarioReportado(): DTOUsuarioReportado {
    return DTOUsuarioReportado(
        id = this.id,
        nombre = this.nombre,
        email = this.email,
        reportado = this.reportado
    )
}